client/my-sites/customer-home/cards/tasks/site-setup-list/get-task.js

```diff
@@ -132,7 +132,9 @@ export const getTask = (
 				description: translate(
 					'Add your store details, add products, configure shipping, so you can begin to collect orders!'
 				),
-				actionText: translate( 'Finish store setup' ),
+				actionText: task.isCompleted
+					? translate( 'WooCommerce Home' )
```

Comment:
I'm not sure but `Go to WooCommerce` might be better.

Reply:
Oh yes, maybe having a verb there would be better

Reply:
![bitmoji](https://sdk.bitmoji.com/render/panel/c7cb9dee-3c6f-488e-96e3-76db13f87c0d-b931236c-c79a-49b5-b0f5-ab26d7e4cd09-v1.png?transparent=1&palette=1&width=30)  I think @arthur791004's suggestion is good, it is what first popped into my mind.  But a small detail so either with or without "Go to" would be fine. :)